modules/containers-signature-verify-application.adoc

```diff
@@ -0,0 +1,251 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-container-signature.adoc
+
+[id="containers-signature-verify-application_{context}"]
+= Verifying the signature verification configuration
+After applying machine configs to the cluster, the machine-config-controller notices the new Machine Config and generate a new version of the worker, such as `rendered-worker-<hash>`.
```

Comment:
I'm not sure if this should be 
"After applying machine configs to the..." or "After the Machine Config Operator (MCO) applies changes to the..." 

@mburke5678 you probably know far more about the MCO than I do. Do you have any suggestions?
https://docs.openshift.com/container-platform/4.7/post_installation_configuration/machine-configuration-tasks.html

Reply:
I used this structure in the logging docs: 
_The controller detects the new `MachineConfig` object and generates a new `rendered-worker-<hash>` version._

I think what you have here works. Although, I like _detects_ better than _notices_. Seems more computer-ish.

Should probably s/machine-config-controller/Machine Config Controller, like [here](https://docs.openshift.com/container-platform/4.7/post_installation_configuration/machine-configuration-tasks.html#create-a-kubeletconfig-crd-to-edit-kubelet-parameters_post-install-machine-configuration-tasks).

Reply:
@EricPonvelle @bmcelvee Did we have a discussion about `oc apply` vs `oc create`? I seem to remember we had a preference of one over the other.

Reply:
Thank you, @mburke5678! I only gave a cursory review, but got hung-up on some of my knowledge gaps in the subject area.